### DISCUSSION TITLE
Would it be possible to add lower charging limits ?

### DISCUSSION THREAD

**Discussion by Yaksi** [seerge/g-helper#454] [Ideas]
I do have a ga502du and using a dongle to charge it with pd usb c , it doesn't have type c charging from the type c port. Anyhow my charger is 65w it is enough to charge or use it on battery save/balance mode but not for both so when i want to use the laptop with the small pd adapter I have to either wait till it has 55 percent. If the charging limit was lower I could have use it as much as i want because it wouldn't have lost battery. Not sure why I kept this so long , i hope it is understandable at least. 

The explanation wasn't exactly necessary, anyway I need to set it at a lower charging limit , I hope that can be achieved as an option right now the lowest is 55.

**Comment by seerge** [comment]
@Yaksi current minimum is 50, not 55. I'm not sure if it's good for battery if you will constantly keep it lower than 50

**Reply by Yaksi** [reply]
True my mistake, it is 50, I don't keep it normally at that percentage it is only when i am outside and need to use the small adapter while also not having enough battery life. If it can be done as an option in the app that would be great honestly.

**Reply by Yaksi** [reply]
![IMG_20230524_154934.jpg](https://github.com/seerge/g-helper/assets/29440331/916aea3d-bf7d-4bff-8e9c-67d0e021a16a)

This is the thing that i use

**Reply by seerge** [reply]
@Yaksi but charge limit, is not a wattage - It's % till what battery will keep charging, and then stop. 
How is it connected with power of your adapter? You can use laptop during charging as much as you want.

**Reply by Yaksi** [reply]
The issue is that if I try to charge at the same time while using , it starts to do a disconnect and reconnect thing  because of not having enough amp. But what I can do is either stop charging and just use it normally or put it on sleep then charge. Can't do both with this 65w pd charger. 

I think it works a bit different than normal type c charging. The computer still thinks it is connected to barrel plug so tries to take more wattage probably.

**Reply by seerge** [reply]
@Yaksi you can try this build then, it should allow you to go down to 40%
[GHelper.zip](https://github.com/seerge/g-helper/files/11556087/GHelper.zip)

**Reply by Yaksi** [reply]
Did tried it, it does work. Thanks a lot appreciate all the good work , tbh for my model asus armoury create did broke  the keyboard lighting, whenever it goes to sleep , backlight stops working , I did report to them and gave logs and everything, the issue still is there and there is no solution other than getting rid of armoury create. Thanks to you with the g-helper no more bloatware and no issues. You have brought life to this laptop.
